libs/atqr/domain/src/lib/valueObjects/notification.ts

```diff
@@ -0,0 +1,113 @@
+import { ValueObject } from '../../utils/valueObject';
+import { Guid } from '@tokilabs/lang';
+import { NotificationLogEntry } from './notificationLogEntry.valueObject';
+import { MessageTemplateId } from '../services';
+import {
+  NotificationChannel,
+  NotificationCategory,
+} from '../types/notification.types';
+export class Notification extends ValueObject<Notification> {
+  constructor(
+    public readonly id: Guid,
```

Comment:
The id needs to be instantiated, correct?

Solving this comment, I believe that @svallory  can now do the last review

Reply:
No, it does not. Value objects may rarely automatically set values (createdAt is an example of it)